.github/workflows/validate.yaml

```
@@ -87,7 +87,7 @@ jobs:
             1>&2 echo "Running with fuseki"
             test_harness+="./with-fuseki.sh"
           fi
-          black --config black.toml --check ./rdflib || true
+          black --config black.toml --check --diff . || true
```

```suggestion
          black --config black.toml --check --diff ./rdflib || true
```

Let's rather keep it as is for now, but ideally we should try and rely solely on config for these tools.